components/tag-list/tag-list-item.js

```diff
@@ -9,12 +10,47 @@ class TagListItem extends TagListItemMixin(LitElement) {
 			 * REQUIRED: Text to display
 			 * @type {string}
 			 */
-			text: { type: String }
+			text: { type: String },
+			/**
+			 * Optional: Text to display in tooltip.
+			 * Tooltip will also include text property value if truncated.
+			 * @type {string}
+			 */
+			description: { type: String },
+			_isTruncated: { attribute: false },
+			_resizeObserver: { attribute: false }
```

Comment:
I wouldn't add the `_resizeObserver` as an property . Might be better to also just do `this.RequestUpdate()` when `_isTruncated` changes and not have it as an property either.

Reply:
Can remove `_resizeObserver`, will leave `_isTruncated` as a property so I don't manually have to manually `this.RequestUpdate()` on changes unless Gaudi folks think this should be done.